clcache.py

```diff
@@ -1334,7 +1334,7 @@ def processDirect(cache, outputFile, compiler, cmdLine, sourceFile):
             origCmdLine = cmdLine
```

Comment:
But isn't the entire purpose of  `origCmdLine` to be able to modify `cmdLine`? It's like a Backup. So it should relyably work no matter *how* we change `cmdLine` later on.

If we went for `cmdLine.insert(0, '/showIncludes')`, the `origCmdLine` would be altered just because we did not do a proper copy.